### ISSUE TITLE
Attempting to create a webhook on production produces the error "Can not create Apex Class on an active organization."

### ISSUE THREAD

**Issue by ugpeter** [jamesward/salesforce-webhook-creator#36]
Hi @jamesward 

Have been using this for many years (via the hosted heroku site) and have had great success.  I recently tried to set up webhooks on a Salesforce instance.  It logged in successfully (to a production site) and allowed me to select the objects and then type in the details.

However, when I attempted to create the webhook, it gave the error "Can not create Apex Class on an active organization.".

Googling this error seems to indicate that you can't change a production site, but I'm not sure how it would apply to using your app.  Can you please suggest if there's an issue within the app or a way around this?

**Comment by jamesward** [comment]
It sounds like there were new restrictions put on the production app.  In this case I believe you'll have to login to the sandbox and then do the usual sandbox -> prod promotion thing.

**Comment by ugpeter** [comment]
Yep - you're right.  I got it working on sandbox then made an outbound change set to push into production.  It's a shame that in building the outbound change set, it was not able to pick up which objects changed.  I had to manually go through the list to work it out.
